tidy3d/components/heat/monitor.py

```diff
@@ -16,6 +17,19 @@ class HeatMonitor(AbstractMonitor, ABC):
 class TemperatureMonitor(HeatMonitor):
     """Temperature monitor."""
 
+    unstructured: bool = pd.Field(
+        False,
+        title="Unstructured Grid",
+        description="Return data on the original unstructured grid.",
+    )
+
+    conformal: bool = pd.Field(
+        False,
+        title="Conformal Monitor Meshing",
+        description="Mesh monitor domain exactly. Note that this option affects the computational "
```

Comment:
Could you elaborate for the user's sake what it means to mesh the domain exactly? So if we have `unstructured=True` and `conformal=False`, I guess it would take the mesh based on the heat domain's mesh, and if `conformal=True` then the heat mesh is adjusted based on the monitor? But if `unstructured=False`, then does `conformal` have an effect?

Reply:
changed description to this
```
description="If ``True`` the heat simulation mesh will conform to the monitor's geometry. "
"While this can be set for both Cartesian and unstructured monitors, it bears higher "
"significance for the latter ones. Effectively, setting ``conformal = True`` for "
"unstructured monitors (``unstructured = True``) ensures that returned temperature values "
"will not be obtained by interpolation during postprocessing but rather directly "
"transferred from the computational grid."
```